src/main/java/core/tasks/Event.java

```diff
@@ -0,0 +1,98 @@
+package core.tasks;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+import core.exceptions.NoDescriptionGivenException;
+import core.exceptions.NoEventLocaleMentionedException;
+import utilities.OutputFormatter;
+
+/**
+ * The Event class.
+ *
+ * @author s7manth
+ * @version 0.1
+ */
+public class Event extends Task {
+    protected LocalDate atDate;
+    protected LocalTime atTime;
+    protected String at;
+
+    /**
+     * Constructor for the Event class.
+     *
+     * @param description The description of the event.
+     * @param at The locale of the event.
+     */
```

Comment:
Similar to "by" for the deadline class, perhaps using the word "locale" like you used in the javadoc comment would be more clearly understood than "at"?